flask_peewee/tests/test_rest.py

```diff
@@ -201,21 +202,21 @@ def test_reverse_resources(self):
         # Test reverse resource serialization
         resp = self.app.get('/api/amodelv2')
         self.assertEqual(resp.get_json(), [
-            {'a_field': 'a1', 'bmodel_set': [{'a': 1, 'b_field': 'b1', 'c': None}], 'id': 1},
-            {'a_field': 'a2', 'bmodel_set': [{'a': 2, 'b_field': 'b2', 'c': None}], 'id': 2},
+            {'a_field': 'a1', 'bmodel_set': [{'a': ANY, 'b_field': 'b1', 'c': None}], 'id': ANY},
+            {'a_field': 'a2', 'bmodel_set': [{'a': ANY, 'b_field': 'b2', 'c': None}], 'id': ANY},
         ])
 
         # Test filter on reverse resources
         resp = self.app.get('/api/amodelv2?bmodel__b_field=b2')
         self.assertEqual(resp.get_json(), [
-            {'a_field': 'a2', 'bmodel_set': [{'a': 2, 'b_field': 'b2', 'c': None}], 'id': 2},
+            {'a_field': 'a2', 'bmodel_set': [{'a': ANY, 'b_field': 'b2', 'c': None}], 'id': ANY},
```

Comment:
Same question regarding the removal of specificity. My concern is that the query botch the join and we wouldn't catch it.